x/bank/keeper/grpc_query_test.go

```diff
@@ -324,7 +324,7 @@ func (suite *IntegrationTestSuite) TestGRPCDenomOwners() {
 
 		bal := sdk.NewCoins(sdk.NewCoin(
 			sdk.DefaultBondDenom,
-			sdk.TokensFromConsensusPower(initialPower/10, sdk.DefaultPowerReduction),
```

Comment:
ideally we don't use tokensfromconsensuspower randomly in tests, or is it random. I couldn't find a reason to use it other than it did a multiplication for you?

Reply:
lol yeah, agreed